src/components/notifications/Keep.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <div class="mb-3">
+        <label for="webhook-url" class="form-label">{{ $t("Host URL") }}</label>
```

Comment:
Please make sure that all translations are included in [`src/lang/en.json`](https://github.com/louislam/uptime-kuma/blob/master/src/lang/en.json).
Only the ones which are can be translated by our awsome team of translators ^^

Reply:
Added 'Host URL' to `src/lang/en.json`